examples/ExampleRunner.jl

```diff
@@ -137,8 +137,8 @@ function create_screens(rootscreen)
         "play.png", "pause.png"
     ]
     imgs = map(paths) do path
-        img = map(RGBA{U8}, loadasset(path))
-        img, flipdim(img, 1)
+        img = convert(Matrix{RGBA{N0f8}}, loadasset(path))
```

Comment:
I explicitely want to force it to be `RGBA{N0f8}`.
I introduced the convert to be backwards compatible to the old Images, which would otherwise return a different storage order!